Frontend/src/Components/GeneralData/Range_Cell.js

```diff
@@ -1,12 +1,12 @@
 import {Flex, Text, Spacer, Box} from "@chakra-ui/react"
 
-export default function CellString(props){
+export default function Range_Cell(props){
     return(
         <Flex flex='2' pl='1' pr='1' direction='column'>
             <Flex flex='0.4'>
-                <Text fontSize='1vw'>{props.label}</Text>
+                <Text fontSize={props.fontSize}>{props.label}</Text>
                 <Spacer/>
-                <Text fontSize='1vw'>{props.temperature.toFixed(3)}&#8451;</Text>
+                <Text fontSize={props.fontSize}>{props.dataType.toFixed(props.digits)}{props.unitType}</Text>
```

Comment:
"dataType" (and less so "unitType") is/are a bit misleading. Consider changing to some thing like "data" and "unit"